tests/installation/licensing/verify_license_translations.pm

```diff
@@ -55,7 +55,7 @@ sub run {
         $license_agreement->select_language($translation->{language});
         $license_agreement_info = $license_agreement->collect_current_license_agreement_info();
         if ($license_agreement_info->{text} !~ /$translation->{text}/) {
-            $errors .= "EULA content for the language: '$translation->{language}' didn't validate. Please, see autoints-log for the detailed content of EULA\n";
+            record_soft_failure("EULA content for the language: '$translation->{language}' didn't validate. This should not happen after BETA!\n");
```

Comment:
this is more complicated than that, there are 3 stages for the eula:
https://confluence.suse.com/display/SUSELinuxEnterpriseServer15SP5/SLE15+SP5+Quality
and in practice I'm been requested some of them until GMC
We don't have that granularity in this test, but it sort of work because we match with a small amount of text.
Therefore it might not match after BETA, and the idea it is not having failing this scenario and been able to adjust the workaround with the missing parts.
- But we still to have a bug open with the language missing stating that for now is expected the Beta EULAS for them.
- @rakoenig what about to create a new field in the test data `available: yes` | `available: no` so we still can track what fails and what we accept which is not there yet?

As we don't have the granularity to distinguish between beta uelas and final eulas in this test, I think it would be more useful to point to the bug that this additional message, and only record soft-failure for those two cases, but if in next build is missing another one, it should fail, we need to adjust it and go to the bug and comment the issue.

Reply:
@jknphy creating new fields in the test data which is part of the yaml would mean, that we need to update this whenever a new translation is available. That looks a bit weird to me.

Reply:
I see what you meant, so we will create a record sof-failure for each one missing, that is fine, perhaps we don't need any inclusion list, just replace 'This should not happen after BETA!' by the a NEW bug reference, because according to that confluence link is not that clear that after beta you have the translations for everything. makes sense to you know?